include/transports/dds/components/RTPSServerProcedureEndpoint.h

```diff
@@ -22,139 +22,151 @@
 
 #include <string>
 
-namespace boost
+namespace boost {
+class mutex;
+} // namespace boost
```

Comment:
Is there a need for a boost mutex, could we use a std::mutex instead?
Are there other boost dependencies?
Can we remove dependency on boost?

Reply:
I agree with you. But we think is out of scope of this PR.